tasks/__init__.py

```diff
@@ -76,10 +76,10 @@ def git_logs():
 
 
 @task
-def apiserver(port=8000, wait=True):
+def apiserver(port=8000, wait=True, host='0.0.0.0'):
```

Comment:
As noted above, 127.0.0.1 won't be sufficient for this system to run, unless the RMap server is moved to the same computer as the OSF server.  But 0.0.0.0 may be more accepting than necessary!